ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/nested-virtualization/provision/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-provision-chain.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/nested-virtualization/provision/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-provision-chain.yaml",
```

Comment:
Actually this steps in this chain is the same as customer vpc, I prefer to rename it to `customer-vpc`

Reply:
Do you mean to replace "nested-virtualization" with "customer-vpc"?

Reply:
right, but once that, need to move the env setting in https://github.com/openshift/release/pull/47059/files#diff-7b5425bc609a27d9e0b8585244d96de4b460cfb98893249cb175ddccf081915fR8 to job config level.

Reply:
Something like:
```
cucushift/installer/rehearse/gcp/ipi/customer-vpc/{provision,deprovision}

cucushift/installer/rehearse/gcp/ipi/nested-virtualization/provision/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-workflow.yaml
- chain: cucushift-installer-rehearse-gcp-ipi-customer-vpc
- ref: cucushift-installer-check-gcp-nested-virtualization
```

Reply:
Keep including the step `ipi-conf-gcp-control-plane-node-type` in `cucushift-installer-rehearse-gcp-ipi-customer-vpc-provision` chain?

Reply:
I think so.